sdk/src/native_loader.rs

```diff
@@ -1,14 +1,24 @@
-use crate::account::{Account, AccountSharedData};
+use crate::account::{
+    Account, AccountSharedData, InheritableAccountFields, DUMMY_INHERITABLE_ACCOUNT_FIELDS,
+};
+use crate::clock::INITIAL_RENT_EPOCH;
 
 crate::declare_id!("NativeLoader1111111111111111111111111111111");
 
 /// Create an executable account with the given shared object name.
 pub fn create_loadable_account(name: &str, lamports: u64) -> AccountSharedData {
+    create_loadable_account_with_fields(name, (lamports, INITIAL_RENT_EPOCH))
+}
+
+pub fn create_loadable_account_with_fields(
+    name: &str,
+    (lamports, rent_epoch): InheritableAccountFields,
```

Comment:
Oh wow!  I didn't realize destructuring could happen in the function args like this